src/utils/common.js

```diff
@@ -71,5 +71,17 @@ export default {
         let contractArr = encodeArr.concat(checkArr)
         let contractString = Base58.encode(contractArr)
         return contractString
+    },
+    getTokenIdByIndex(contraId, tokenIndex) {
```

Comment:
This function is similar with contractIDToTokenID. Just change that function to `contractIDToTokenID(contractId, tokenIndex=0)` should be OK.

Reply:
ok